apps/app/src/errors/messages.ts

```diff
@@ -0,0 +1,3 @@
+export enum error {
+  Something_Wrong = "Something went wrong, please try again",
```

Comment:
I think you'll end up with a lot of different error messages. Hopefully they are very specific to those specific errors (helps with debugging), so keeping them in one file for re-usability should not make sense.

Reply:
I was thinking more in for different languages cases.